src/WebJobs.Script.WebHost/Management/VirtualFileSystem.cs

```diff
@@ -638,15 +645,38 @@ private IEnumerable<VfsStatEntry> GetDirectoryResponse(HttpRequest request, File
             //}
         }
 
-        protected HttpResponseMessage CreateResponse(HttpStatusCode statusCode, object payload = null)
+        internal static HttpResponseMessage CreateResponse(HttpStatusCode statusCode, object payload = null, string mediaType = null)
         {
             var response = new HttpResponseMessage(statusCode);
             if (payload != null)
             {
                 var content = payload is string ? payload as string : JsonConvert.SerializeObject(payload);
-                response.Content = new StringContent(content, Encoding.UTF8, "application/json");
+                mediaType = mediaType ?? "application/json";
+                response.Content = new StringContent(content, Encoding.UTF8, mediaType);
             }
             return response;
         }
+
+        internal static HttpResponseMessage CreateNotFoundResponse(string fileName)
+        {
+            return CreateResponse(HttpStatusCode.NotFound, $"'{fileName}' not found.", "text/plain");
```

Comment:
Noticed that for the existing 404 string responses, the media type was incorrect and the returned string content wasn't correct.  E.g. rather than returning `'C:\code\Git\Azure\azure-webjobs-sdk-script\sample\csharp\blah.txt' not found.` it currently returns `Unexpected '''`.

Reply:
This will create a response without following content negotiation. Is that what you expect?